doc/newsfragments/2481_changed_result.fail_parameters.rst

```diff
@@ -0,0 +1 @@
+Make :py:meth:`fail <testplan.testing.multitest.result.Result.fail>` paramaters consistent with :py:meth:`log <testplan.testing.multitest.result.Result.log>`. **breaking change** if assertion was called previously as `result.fail(description=...)`. Simply ensure message is passed as it became a positional argument.
```

Comment:
Make :py:meth:`result.fail <testplan.testing.multitest.result.Result.fail>` parameters consistent with :py:meth:`result.log <testplan.testing.multitest.result.Result.log>`, that ``message`` is the required argument and ``description`` is optional.